Tests/js/views/assets/ez-actionbarview-tests.js

```diff
@@ -37,49 +37,60 @@ YUI.add('ez-actionbarview-tests', function (Y) {
             this.view.destroy();
         },
 
-        "Should instantiate createContentActionView with contentType": function () {
-            var createContentActionFound = false;
+        _actionAssertions: function (testedActionId) {
+            var actionFound = false;
 
             for (var i = 0; i < this.view.get('actionsList').length; i++){
-                if (this.view.get('actionsList')[i].get('actionId') === 'createContent'){
-                    createContentActionFound = true;
-                    Assert.areSame(
-                        this.view.get('contentType'),
-                        this.view.get('actionsList')[i].get('contentType'),
-                        'The contentType should have been set to createContentActionView'
-                    );
+                if (this.view.get('actionsList')[i].get('actionId') === testedActionId){
+                    actionFound = true;
+
+                    switch ( testedActionId ) {
```

Comment:
not generic, instead the assert method should take in second argument the attributes if should check, something like:
```js
this._isActionCreated('translate', {
    location: this.view.get('location'),
    content: this.view.get('content'),
});
```